### ISSUE TITLE
Specifying timezone causes no trigger using croner <= 7.x

### ISSUE THREAD

**Issue by jefbarn** [Hexagon/croner#225]
This doesn't do anything:
```typescript
  Cron('* * * * * *', { timezone: 'America/Anchorage' }, () => {
    console.log('test')
  })
```

This works:
```typescript
  Cron('* * * * * *', () => {
    console.log('test')
  })
```

**Comment by Hexagon** [comment]
Interersting, can confirm with multiple timezones in the range UTC-9 to UTC-12.

**Comment by Hexagon** [comment]
Released in stable channel through `8.0.0`

**Comment by victorird** [comment]
Thanks for the fix @Hexagon . However I tried the 7.0.6-dev.0 branch with Node.js 16 and it throws the following error:

```
RangeError: Value shortOffset out of range for Date.prototype.toLocaleString options property timeZoneName
    at Date.toLocaleString (<anonymous>)
    at getTimezoneOffset (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:1284)
    at Function.minitz.fromTZ (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:359)
    at minitz (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:75)
    at CronDate.getDate (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:17226)
    at CronDate.getTime (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:17361)
    at Cron.msToNext (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:20340)
    at Cron.schedule (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:20975)
    at new Cron (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:19159)
    at Cron (/home/vagrant/projects/itheia-app/api/node_modules/croner/dist/croner.min.cjs:1:17825)
```

I'm guessing the syntax in 7.0.6-dev.0 is only supported for Node.js >= 18. We're unable to move to Node.js 18 at the moment due to hardware vendor limitation. Is there any chance you can make this bugfix work with Node.js 16? Thx again.

**Comment by Hexagon** [comment]
Oh! Released a potential hot fix in `croner@7.0.6-dev.1`, where i added the following code as a quick workaround for AKDT and AKST not being supported by Date.parse():

```js
	// Convert non-supported timezones
	// - Hotfix for AKST and AKDT 7.0-branch of Croner, fixed for real in 8.0
	tz = tz.replace("AKST","-0900").replace("AKDT","-0800");
```

Hacky, but i hope it should do for those who cannot go to `8.x`

**Comment by victorird** [comment]
> @victorird Oh, thanks for reporting this! Released a potential hot fix in `croner@7.0.6-dev.1` (channel `croner@legacy-node-support`), where i added the following code as a quick workaround for AKDT and AKST not being supported by Date.parse():
> 
> ```js
> 	// Convert non-supported timezones
> 	// - Hotfix for AKST and AKDT 7.0-branch of Croner, fixed for real in 8.0
> 	tz = tz.replace("AKST","-0900").replace("AKDT","-0800");
> ```
> 
> Hacky, but i hope it should do for those who cannot go to `8.x`. Please let me know if this works for you, and i'll release `7.0.6` stable.

Thanks for the quick hot fix. I tested it and it's working flawlessly!

**Comment by Hexagon** [comment]
Nice! Added support for AST and released `croner@7.0.6-dev.2`, will let this version linger for a while before releasing the stable version.

**Comment by Hexagon** [comment]
Resolved in the 7.x branch through `7.0.7`
